sympy/printing/pretty/pretty.py

```diff
@@ -531,7 +531,14 @@ def adjust(s, wid=None, how='<^>'):
         for lim in expr.limits:
             if len(lim) == 3:
                 prettyUpper = self._print(lim[2])
-                prettyLower = self._print(Equality(lim[0], lim[1]))
+
+                var = lim[0].name
+
+                if lim[1] is S.NegativeInfinity:
+                    prettyLower = self._print(var + " = " + str(lim[1]))
```

Comment:
Do not use str in the printers. You need to recursively call self._print to use the same printer.